1_fetch.R

```diff
@@ -0,0 +1,58 @@
+source("1_fetch/src/get_nwis_data.R")
+
+nwis_Cd <- "00010"
+start_date <- "2014-05-01"
+end_date <- "2015-05-01"
```

Comment:
Good call pulling these phase-specific parameters into the phase-specific .R files where the targets that use them are defined.